core/src/main/java/com/rafaskoberg/gdx/parrot/sfx/SoundPlayerImpl.java

```diff
@@ -143,8 +143,13 @@ public void updateSounds(float delta) {
                 // Calculate distance factor
                 tmpVec.set(soundInstance.positionX, soundInstance.positionY).sub(listenerPosition.x, listenerPosition.y);
                 float dst = tmpVec.len();
-                float dstFactorRaw = MathUtils.clamp(dst / settings.soundDistanceLimit, 0.0f, 1.0f);
-                distanceFactor = Interpolation.linear.apply(1.0f, ParrotUtils.dbToVolume(settings.soundDistanceReduction), dstFactorRaw);
+                if(dst > settings.distanceRolloffMinDistance)
+                {
+                    float dstFactorRaw = MathUtils.clamp((dst - settings.distanceRolloffMinDistance) / settings.distanceRolloffMaxDistance, 0.0f, 1.0f);
+                    distanceFactor = settings.distanceRolloffCurve.apply(settings.distanceRolloffMinGain, 1, 1 - dstFactorRaw);
+                }
+                else
+                    distanceFactor = 1;
```

Comment:
Just some formatting here hehe
```suggestion
                if(dst > settings.distanceRolloffMinDistance) {
                    float dstFactorRaw = MathUtils.clamp((dst - settings.distanceRolloffMinDistance) / settings.distanceRolloffMaxDistance, 0.0f, 1.0f);
                    distanceFactor = settings.distanceRolloffCurve.apply(settings.distanceRolloffMinGain, 1, 1 - dstFactorRaw);
                } else {
                    distanceFactor = 1;
                }
```